doc/website/getting-started/list-of-examples.md

```diff
@@ -0,0 +1,16 @@
+# List of examples
+
+| Example                                         | Description                                                 | Level              |
```

Comment:
Please order the list alphabetically.

Reply:
Alphabetically won't help new readers, the table is structured by easy/beginner/important ... hard/advanced/unimportant